pkg/apis/core/types_shoot.go

```diff
@@ -189,6 +191,20 @@ type NetworkingStatus struct {
 	EgressCIDRs []string
 }
 
+// InPlaceUpdatesStatus contains information about in-place updates for the Shoot workers.
+type InPlaceUpdatesStatus struct {
+	// PendingWorkersRollouts contains information about worker pools pending in-place updates.
+	PendingWorkersRollouts *InPlaceUpdatePendingWorkers
```

Comment:
Is `Rollouts` the appropriate suffix here? Maybe just naming the variable the same as the type is ok or alternatively like this:
```suggestion
	PendingInPlaceWorkerUpdates *InPlaceUpdatePendingWorkers
```

Reply:
I was just trying to use the same name as 
https://github.com/gardener/gardener/blob/6bf5aecc578a472c1fca2babeb6b8317ba9535b7/pkg/apis/core/types_shoot.go#L247-L249
But maybe this suggestion is also okay.

Reply:
How about just `PendingWorkerUpdates`? The parent json field has `InPlace` in it and so does the child.